src/main/java/io/aiven/kafka/connect/gcs/GcsSinkTask.java

```diff
@@ -107,6 +107,7 @@ private void flushFile(final String filename, final List<SinkRecord> records) {
                      .withExternalProperties(config.originalsStrings())
                      .withOutputFields(config.getOutputFields())
                      .withCompressionType(config.getCompressionType())
+                     .withEnvelopeEnabled(config.envelopeEnabled())
```

Comment:
missed it, for such purpose  `withExternalProperties` has been introduced. but we can change it later